.travis.yml

```diff
@@ -1,21 +1,23 @@
+---
+
 language: go
 go:
-- 1.12.x
-- master
+  - 1.12.x
```

Comment:
`1.13.X` ?

Reply:
Nop, terraform is not compatible with 1.13

Reply:
ok